model-integration/src/main/java/ai/vespa/llm/clients/LocalLLM.java

```diff
@@ -62,46 +65,47 @@ public LocalLLM(LlmLocalClientConfig config) {
                 .setNThreads(config.threads() <= 0 ? defaultThreadCount : config.threads())
                 .setNCtx(config.contextSize())
                 .setNGpuLayers(config.useGpu() ? config.gpuLayers() : 0);
-
+        
+        if (config.seed() != -1)
+            modelParams.setSeed(config.seed());    
+        
+        // Load model
         long startLoad = System.nanoTime();
         model = new LlamaModel(modelParams);
         long loadTime = System.nanoTime() - startLoad;
-        logger.info(String.format("Loaded model %s in %.2f sec", modelFile, (loadTime*1.0/1000000000)));
-
-        // Todo: handle prompt context size - such as give a warning when prompt exceeds context size
-        contextSize = config.contextSize();
-    }
+        logger.fine(() -> String.format("Loaded model %s in %.2f sec", modelFile, (loadTime*1.0/1000000000)));
```

Comment:
Consider using `Duration` in the future for time calculation. This have earlier been a typical source for bugs (although not that important here as this is just for logging).